.pre-commit-config.yaml

```diff
@@ -121,3 +125,4 @@ ci:
     - bandit
     - mypy
     - gitleaks
+    - vale
```

Comment:
FYI @maxschulz-COL @stichbury this should have been on the skip list for https://pre-commit.ci/ already, because vale just flags errors rather than fixing them. The only things we run on pre-commit.ci are things that can fix stuff.